### PR TITLE
New formula: cwltest

### PR DESCRIPTION
**Pull request by roryk**
* [x] I have read the guidelines above.
* [x] This PR adds a new recipe.
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

This is necessary to upgrade to the more recent versions of cwltool.

https://github.com/common-workflow-language/cwltest

Pinging @bioconda/core for a review. Thanks ya'll!

### CODE REVIEW

File: recipes/cwltest/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+requirements:
+  build:
+    - python
+    - setuptools
+    - schema-salad ==1.14.20160708181155
+    - typing
+
+  run:
+    - python
+    - setuptools
```
Comment (bgruening) [inline recipes/cwltest/meta.yaml]: Do we need setuptools as run time dependency?

Comment (johanneskoester) [inline recipes/cwltest/meta.yaml]: Sometimes packages depend on certain setuptools modules to find resources. But I don't know if it is the case here. @roryk are you aware of that?

Comment (roryk) [inline recipes/cwltest/meta.yaml]: Thanks, it doesn't need setuptools to run, afaik, so I dropped it from the requirement.

**Comment by roryk** [comment]
Thanks!

**Pull request merged**